rllib/agents/mbmpo/mbmpo_torch_policy.py

```diff
@@ -0,0 +1,63 @@
+import logging
+
+import ray
+from ray.rllib.policy.torch_policy_template import build_torch_policy
+from ray.rllib.agents.ppo.ppo_tf_policy import postprocess_ppo_gae, \
+    setup_config
+from ray.rllib.agents.ppo.ppo_torch_policy import vf_preds_fetches
+from ray.rllib.agents.a3c.a3c_torch_policy import apply_grad_clipping
+from ray.rllib.utils.framework import try_import_torch
+from ray.rllib.models.catalog import ModelCatalog
+from ray.rllib.agents.maml.maml_torch_policy import setup_mixins, \
+    maml_loss, maml_stats, maml_optimizer_fn, KLCoeffMixin
+
+torch, nn = try_import_torch()
+
+logger = logging.getLogger(__name__)
+
+
+def make_model_and_action_dist(policy, obs_space, action_space, config):
+    # Get the output distribution class for predicting rewards and next-obs.
+    policy.distr_cls_next_obs, num_outputs = ModelCatalog.get_action_dist(
+        obs_space, config, dist_type="deterministic", framework="torch")
+
+    # Build one dynamics model if we are a Worker.
+    # If we are the main MAML learner, build n (num_workers) dynamics Models
+    # for being able to create checkpoints for the current state of training.
+    policy.dynamics_model = ModelCatalog.get_model_v2(
+        obs_space,
+        action_space,
+        num_outputs=num_outputs,
+        model_config=config["dynamics_model"],
+        framework="torch",
+        name="dynamics_ensemble",
+    )
+
+    action_dist, num_outputs = ModelCatalog.get_action_dist(
```

Comment:
Actually, you can skip this now and just use `build_torch_policy(make_model=...)` (w/o the `_and_action_dist`, iff the action dist is a default one anyways.